go.mod

```diff
@@ -148,6 +148,7 @@ replace (
 	k8s.io/controller-manager => k8s.io/controller-manager v0.27.3
 	k8s.io/cri-api => k8s.io/cri-api v0.27.3
 	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.27.3
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.27.3
```

Comment:
I think `release-notes` is failing because this line is missing. I'll run it again to generate the 2.6.0 release notes